src/font.ts

```diff
@@ -129,8 +117,8 @@ export class Font {
       family = f;
     }
 
-    family = family ?? Font.SANS_SERIF;
-    size = size ?? Font.SIZE + 'pt';
+    family = family ?? '';
+    size = size ?? '';
```

Comment:
I suggest we hardcode something here as a fallback, if you want to remove the constants.

This `validate()` method is supposed to return a valid font object that makes sense in the CSS world.

So we should put 'sans-serif' as the fallback family, and '10pt' or '12pt' as the fallback font size, so that this method doesn't return empty strings for those fields.

Reply:
done, use configured one